pkg/gather/clusterconfig/sap_license_management_logs.go

```diff
@@ -0,0 +1,100 @@
+package clusterconfig
+
+import (
+	"context"
+
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/kubernetes"
+	corev1client "k8s.io/client-go/kubernetes/typed/core/v1"
+
+	"github.com/openshift/insights-operator/pkg/record"
+)
+
+// GatherSAPLicenseManagementLogs collects logs from license management pods with the following substrings:
+//   - "can't initialize iptables table",
+//
+// The Kubernetes API https://github.com/kubernetes/client-go/blob/master/kubernetes/typed/core/v1/pod_expansion.go#L48
+// Response see https://docs.openshift.com/container-platform/4.6/rest_api/workloads_apis/pod-core-v1.html#apiv1namespacesnamespacepodsnamelog
+//
+// Location in archive: config/pod/sdi/logs/{pod-name}/errors.log
+func GatherSAPLicenseManagementLogs(g *Gatherer, c chan<- gatherResult) {
+	defer close(c)
+
+	messagesToSearch := []string{
+		"can't initialize iptables table",
+	}
+
+	dynamicClient, err := dynamic.NewForConfig(g.gatherKubeConfig)
+	if err != nil {
+		c <- gatherResult{errors: []error{err}}
+		return
+	}
+
+	kubeClient, err := kubernetes.NewForConfig(g.gatherProtoKubeConfig)
+	if err != nil {
+		c <- gatherResult{nil, []error{err}}
+		return
+	}
+
+	coreClient := kubeClient.CoreV1()
+
+	records, errs := gatherSAPLicenseManagementLogs(g.ctx, dynamicClient, coreClient, messagesToSearch)
+	c <- gatherResult{records, errs}
+}
+
+func gatherSAPLicenseManagementLogs(
+	ctx context.Context,
+	dynamicClient dynamic.Interface,
+	coreClient corev1client.CoreV1Interface,
+	messagesToSearch []string,
+) ([]record.Record, []error) {
+	datahubsResource := schema.GroupVersionResource{
+		Group: "installers.datahub.sap.com", Version: "v1alpha1", Resource: "datahubs",
```

Comment:
One idea. I know we are doing it just like here, but when there will be a new SAP version available then it will be a bit pain to find all the occurences. What about to extract the GVR to some more widely shared constant?

Reply:
Yeah, the function makes sense. Do you think we should put it in this PR or maybe in some refactoring later?

Reply:
@Sergey1011010 Either way is fine with me, but I'd say it may be easier to just refactor it later for backporting consistency.